config/initializers/devise.rb

```diff
@@ -239,6 +239,8 @@
   # Add a new OmniAuth provider. Check the wiki for more information on setting
   # up on your models and hooks.
   # config.omniauth :github, 'APP_ID', 'APP_SECRET', scope: 'user,public_repo'
+  config.omniauth :facebook, ENV["FACEBOOK_CLIENT_ID"], ENV["FACEBOOK_CLIENT_SECRET"]
```

Comment:
I haven't set up the keys yet. I wanted to configure the basic Omniauth setup asap and I figured we could discuss it either in Slack or in the team meeting on Monday. 

I figured we could use Figaro and set Heroku with the keys, like we have in previous FHP apps, but again I wanted to discuss it with the group first.